### PR TITLE
Update test cases with equivalence partitioning and boundary values

### LINKED ISSUE #183: Improve testing by using equivalence partitions and boundary values
Specify what parts of test cases have inputs from specific equivalence partitions where possible

### PR DESCRIPTION
**Pull request by curtischang2510**
Resolve #183 
- Revised test cases to incorporate equivalence partitioning and boundary value analysis.
- Improves coverage by testing representative cases and edge scenarios for increased reliability and robustness.

### CODE REVIEW

File: src/test/java/tuteez/logic/commands/AddCommandTest.java [hunk 1]
```diff
@@ -62,13 +62,40 @@ public void execute_duplicatePerson_throwsCommandException() {
     }
 
     @Test
-    public void execute_clashingWithAnotherStudentLesson_throwsCommandException() throws CommandException {
+    public void execute_clashingWithAnotherStudentLesson_throwsCommandException() {
```
Comment (weijianwong) [inline src/test/java/tuteez/logic/commands/AddCommandTest.java:65]: Nice work identifying all the EP!
